.github/workflows/ci.yml

```diff
@@ -9,18 +9,19 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [18, 20]
 
     steps:
     - name: Checkout
       uses: actions/checkout@v4
       with:
         fetch-depth: 0
-    - name: Setup Nodejs Env
-      run: echo "NODE_VER=`cat .nvmrc`" >> $GITHUB_ENV
     - name: Setup Nodejs
-      uses: actions/setup-node@v3
+      uses: actions/setup-node@v4
       with:
-        node-version: ${{ env.NODE_VER }}
+        node-version: ${{ matrix.node }}
```

Comment:
If we move away from the matrix, I'd love to switch everything over to using `node-version-file` instead https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file (example from paragon's release workflow: https://github.com/openedx/paragon/blob/98e6313e923c2a70b199f358c2a7063f03647a36/.github/workflows/release.yml#L19)

Reply:
This is sorta what we have now, except it doesn't use that particular setup-nodeism.  No objections to moving to that after we remove the matrix, though.